framework/meta/src/tools/find_workspace.rs

```diff
@@ -0,0 +1,32 @@
+use colored::Colorize;
+use std::path::{Path, PathBuf};
+
+/// Finds the workspace by searching for the workspace argument into the project's cargo.
+/// Works in debug mode too.
+///
+pub fn find_workspace() -> PathBuf {
```

Comment:
`find_workspace() -> Option<PathBuf>`

make it recover in case there is no workspace, more versatile this way

I would also split in 2:
find_current_workspace()
find_workspace(path)

(You can give the current path to commands)

Would also be interesting to see if we can write an integration test in meta. It should find the workspace 2 levels up.